1590 parent child indication

JarrodBaker opened the pull request:
- Updated `approvedIndication` to come from the Indication index rather than Drug (where it no longer exists). 
- Fixed test inputs so proper static file is used for input generation.

JarrodBaker committed:
Update approvedIndications to come from Indication index

Resolves opentargets/platform#1590

JarrodBaker committed:
Add drug inputs static file to Drug ID generator.

mkarmona commented on app/models/gql/Objects.scala:
```diff
@@ -726,6 +725,15 @@ object Objects extends Logging {
+        OptionType(ListType(StringType)),
+        description = Some("Indications for which there is a phase IV clinical trial"),
+        resolve = c => {
+          val indications = DeferredValue(indicationFetcher.defer(c.value.id))
```
@JarrodBaker Should it be coded explicitly The optional list returned with `deferOpt` instead `defer`?

JarrodBaker committed:
Fix indication to return optional for approved Indications

- resolves comment from @mkarmona.

mkarmona committed:
refactor: approvedIndications is either a null None or a list of approved indications Some(...) and None is an algebraic case object and it is always the same no matter the content of the some so there is no need to capture.

Also, being conformant to other endpoints like indications we either null or have something valid. Still something not fully homogeneous across the whole API though.

mkarmona committed:
fix: better to flatMap so if approvedIndications comes as null
